MapboxNavigation/Locale.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+extension Locale {
+    static var preferredLocalLanguageCountryCode: String {
+        let bundleLanguage = Bundle.main.preferredLocalizations.first!
+        let bundleLanguages = bundleLanguage.components(separatedBy: "-")
```

Comment:
This isn’t a list of languages, it’s a list of components detailing a single locale. See [this article](https://en.wikipedia.org/wiki/IETF_language_tag) for more information.